src/gui/socketapi/socketapi.cpp

```diff
@@ -1367,7 +1367,7 @@ void SocketApi::command_ASYNC_ASSERT_ICON_IS_EQUAL(const QSharedPointer<SocketAp
     if (value.name() == iconName) {
         job->resolve();
     } else {
-        job->reject("iconName " + iconName + " does not match: " + value.name());
+        job->reject(QStringLiteral("iconName ") + iconName + QStringLiteral(" does not match: ") + value.name());
```

Comment:
it could use .arg()

Reply:
it *should* use `.arg()`. If it's a user-visible string, use `tr()`.